Doc/library/pathlib.rst

```diff
@@ -462,6 +462,20 @@ Pure paths provide the following methods and properties:
       True
 
 
+.. method:: PurePath.is_relative_to(*other)
```

Comment:
I think to name this parameter as `other_path` would be more explicit.

Reply:
Some other functions use `*other` too, such as: PurePath.relative_to(*other), PurePath.joinpath(*other). So I think keep consistency is ok. If most people like use `other_path`, i would update it.

Reply:
Some other functions use `*other` too, such as: PurePath.relative_to(*other), PurePath.joinpath(*other). So I think keep consistency is ok. If most people like use `other_path`, i would update it.

Reply:
Yes, I just checked `PurePath.relative_to` & `PurePath.joinpath`. In that case, it seems we're good on this part.